handlers/movieHandler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/RealImage/moviebuff-sdk-go"
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+
+	"github.com/ashwinspg/explore-golang/config"
+)
+
+//GetMovieHandler - get movie details
+func GetMovieHandler(w http.ResponseWriter, r *http.Request) {
+	UUID := chi.URLParam(r, "id")
```

Comment:
Use lowercase for local variables https://github.com/golang/go/wiki/CodeReviewComments#variable-names
```suggestion
	uuid := chi.URLParam(r, "id")
```

Reply:
Sure, I'll change it.